examples/src/main/java/org/apache/spark/examples/ml/JavaKMeansExample.java

```diff
@@ -74,6 +76,7 @@ public static void main(String[] args) {
     JavaSparkContext jsc = new JavaSparkContext(conf);
     SQLContext sqlContext = new SQLContext(jsc);
 
+    // $example on$
```

Comment:
I would exclude less relevant imports such as SparkConf, Pattern, SQLContext, etc.  (Only include ones which appear in the example section.)